modules/oe_theme_content_project/templates/field--node--extra-field-oe-theme-content-project-funding--oe-project.html.twig

```diff
@@ -0,0 +1,48 @@
+{#
+/**
+ * @file
+ * Theme override for Funding extra field plugin.
+ *
+ * Available variables:
```

Comment:
Instead of "Available variables" ` * @see ./core/themes/stable/templates/field/field.html.twig` can be used. See https://github.com/openeuropa/oe_theme/pull/507/files